src/index.js

```diff
@@ -238,51 +243,57 @@ const hiddenSearchFields = ['address', 'accepting', 'notAccepting', 'notes', 'se
 
 // create an item for the side pane using a location
 const createListItem = (location, status, lng, lat) => {
-  const urgentNeed = location.urgentNeed ? `<p class="urgentNeed p location-list--important"><span data-translation-id="urgent_need">Urgent Need</span>: ${location.urgentNeed}</p>` : ''
+  const neighborhood = location.neighborhood ? `<h3 class="card-subtitle neighborhood">${location.neighborhood}</h3>` : '';
+
+  const urgentNeed = location.urgentNeed ? `<p class="urgentNeed p card-urgent-need"><span data-translation-id="urgent_need">Urgent Need</span>: ${location.urgentNeed}</p>` : ''
```

Comment:
is this paragraph tag being opened & closed where intended?

Reply:
Oh, nevermind, I was reading the colon in the wrong place 🥳

Reply:
No prob! Another place where formatting could improve readability :)